pkg/commands/workload.go

```diff
@@ -616,21 +618,56 @@ func (opts *WorkloadOptions) Create(ctx context.Context, c *cli.Config, workload
 func (opts *WorkloadOptions) LoadInputWorkload(input io.Reader, workload *cartov1alpha1.Workload) error {
 	var in io.Reader
 
-	f, err := os.Open(opts.FilePath)
-	in = f
-	if f == nil && opts.FilePath == "-" {
+	isURL, err := isUrl(opts.FilePath)
+	if err != nil {
+		return fmt.Errorf("unable to check if filepath %q is a valid url: %w", opts.FilePath, err)
+	}
+	switch {
+	case isURL:
+		in, err = opts.getUrlFileContent()
```

Comment:
**Question:** `isUrl(string)` returns `true` when the url points to a resource from a`ftp`, `smb`, etc (i.e. `ftp://myserver/my-sub/my-workload.yaml`) server, does `WorkloadOptions.getUrlFileContent()` suports retrieving content usingthosep protocols?

Reply:
Let's plan to support just http/https for the scope of this GH issue. We can add support for other protocols in the future if there is an ask for it.